ts/src/btcturk.ts

```diff
@@ -655,20 +655,20 @@ export default class btcturk extends Exchange {
 
     parseOHLCVs (ohlcvs, market = undefined, timeframe = '1m', since: Int = undefined, limit: Int = undefined, tail: Bool = false) {
         const results = [];
-        const timestamp = this.safeValue (ohlcvs, 't');
-        const high = this.safeValue (ohlcvs, 'h');
-        const open = this.safeValue (ohlcvs, 'o');
-        const low = this.safeValue (ohlcvs, 'l');
-        const close = this.safeValue (ohlcvs, 'c');
-        const volume = this.safeValue (ohlcvs, 'v');
+        const timestamp = this.safeList (ohlcvs, 't', []);
+        const high = this.safeList (ohlcvs, 'h', []);
+        const open = this.safeList (ohlcvs, 'o', []);
+        const low = this.safeList (ohlcvs, 'l', []);
+        const close = this.safeList (ohlcvs, 'c', []);
+        const volume = this.safeList (ohlcvs, 'v', []);
         for (let i = 0; i < timestamp.length; i++) {
             const ohlcv: Dict = {
-                'timestamp': this.safeValue (timestamp, i),
-                'high': this.safeValue (high, i),
-                'open': this.safeValue (open, i),
-                'low': this.safeValue (low, i),
-                'close': this.safeValue (close, i),
-                'volume': this.safeValue (volume, i),
+                'timestamp': this.safeList (timestamp, i),
```

Comment:
There should be safeInteger and safeNumber to high, open, low etc...

Reply:
Fixed it, thx